### PR TITLE
feat(FEC-11564): Related List

### PR DESCRIPTION
**Pull request by SivanA-Kaltura**
### Description of the Changes

- Add related list component
- Allow related list configuration - position and expandMode (like in playlist)
- Add button to toggle related list
- Add next entry preview on playback end, visible when related list is open and autocontinue=true
- Remove ks parameter from plugin config

Resolves FEC-11564, FEC-12582, FEC-12812

Figma mockups are linked in the first ticket

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment
- [x] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,11 +40,17 @@
...
-    "kaltura-player-js": "https://github.com/kaltura/kaltura-player-js.git#master"
+    "kaltura-player-js": "https://github.com/kaltura/kaltura-player-js.git#3.8.0"
   },
   "publishConfig": {
     "access": "public"
   },
+  "kaltura": {
+    "name": "related",
+    "dependencies": {
+      "playkit-ui-managers": "1.3.0"
```
Comment (giladna) [inline package.json:51]: @SivanA-Kaltura currently it is 1.3.1
I think it may change to 1.3.2 after 1/1 release

Comment (SivanA-Kaltura) [reply package.json:51]: then we'll need to update the version in every dependent project

Comment (SivanA-Kaltura) [reply package.json:51]: you're right, i added the dependencies before the release

File: src/components/close-button/close-button.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const CloseButton = withText({
+  closeText: 'overlay.close'
+})(({onClick, closeText}: {closeText: string; onClick: () => void}) => {
+  return (
+    <div>
+      <Tooltip label={closeText}>
+        <button aria-label={closeText} tabIndex={0} className={`${KalturaPlayer.ui.style.controlButton}`} onClick={onClick}>
+          <Icon
+            activeColor={KalturaPlayer.ui.style.white}
```
Comment (JonathanTGold) [inline src/components/close-button/close-button.tsx:13]: Redundant - the white is the default

File: src/components/close-button/close-button.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            id={`related-close-icon`}
+            path={IconPath.CLOSE}
+            state={IconState.ACTIVE}
```
Comment (JonathanTGold) [inline src/components/close-button/close-button.tsx:16]: Wrong - active is brand color

File: src/components/list-toggle-button/list-toggle-button.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <button
+      aria-label={relatedVideosText}
+      tabIndex={0}
+      disabled={disabled}
+      className={`${styles.listToggleButton} ${KalturaPlayer.ui.style.controlButton} ${KalturaPlayer.ui.style.upperBarIcon} ${
+        active ? styles.active : ''
+      }`}>
+      <Icon
+        activeColor={KalturaPlayer.ui.style.white}
```
Comment (JonathanTGold) [inline src/components/list-toggle-button/list-toggle-button.tsx:19]: activeColor and state props redundant

File: src/components/pagination-arrow/pagination-arrow.tsx [hunk 5]
```diff
@@ -36,7 +31,7 @@ const PaginationArrow = withText({
         activeColor={KalturaPlayer.ui.style.white}
```
Comment (JonathanTGold) [inline src/components/pagination-arrow/pagination-arrow.tsx:31]: activeColor and state props redundant

**Review by JonathanTGold — CHANGES_REQUESTED** [review]

File: src/components/list-toggle-button/list-toggle-button.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+const ListToggleButton = withText({
+  relatedVideosText: 'related.relatedVideos'
+})(({active, disabled, relatedVideosText}: {active: boolean; disabled: boolean; relatedVideosText: string}) => {
+  return (
+    <button
+      aria-label={relatedVideosText}
+      tabIndex={0}
+      disabled={disabled}
+      className={`${styles.listToggleButton} ${KalturaPlayer.ui.style.controlButton} ${KalturaPlayer.ui.style.upperBarIcon} ${
```
Comment (JonathanTGold) [inline src/components/list-toggle-button/list-toggle-button.tsx:15]: why use use both classes controlButton and upperBarIcon They have two different purposes

**Review by JonathanTGold — CHANGES_REQUESTED** [review]

**Review by JonathanTGold — APPROVED** [review]

**Pull request merged**